pytorch_tabular/tabular_model.py

```diff
@@ -20,7 +20,6 @@
 from sklearn.base import TransformerMixin
 from torch import nn
 from tqdm.autonotebook import tqdm
-from pytorch_tabular import config
```

Comment:
do we need this import?

Reply:
According to my IDE this import is never used, because below there is 
`from pytorch_tabular.config import (
    DataConfig,
    ExperimentConfig,
    ExperimentRunManager,
    ModelConfig,
    OptimizerConfig,
    TrainerConfig,
)`
and in rest of the code the variable config is override by read some config file or receiving the config as argument of a function

Reply:
cool.. then we can remove this import